mitosheet/setup.py

```diff
@@ -143,6 +143,7 @@
             # snowflake-connect-python requires at least Python 3.7
             'snowflake-connector-python[pandas]; python_version>="3.7"',
             'streamlit>=1.24',
+            'dash'
```

Comment:
We should keep an eye out for version requirements of dash as we go forward. But this is fine for demoes